crates/rslint_core/src/groups/errors/no_confusing_arrow.rs

```diff
@@ -0,0 +1,115 @@
+use crate::rule_prelude::*;
+use ast::{ArrowExpr, Expr};
+
+declare_lint! {
+    /**
+    Disallow arrow functions where they could be confused with comparisons.
+
+    Arrow functions (`=>`) are similar in syntax to some comparison operators (`>`, `<`, `<=`, and `>=`).
+    This rule warns against using the arrow function syntax in places where it could be confused with
+    a comparison operator
+
+    Here's an example where the usage of `=>` could be confusing:
+
+    ```js
+    // The intent is not clear
+    var x = a => 1 ? 2 : 3;
+    // Did the author mean this
+    var x = function (a) { return 1 ? 2 : 3 };
+    // Or this
+    var x = a >= 1 ? 2 : 3;
+    ```
+
+    ## Incorrect Code Examples
+
+    ```js
+    var x = a => 1 ? 2 : 3;
+    var x = (a) => 1 ? 2 : 3;
+    ```
+    */
+    #[serde(default)]
+    NoConfusingArrow,
+    errors,
+    "no-confusing-arrow",
+    /// Relaxes the rule and accepts parenthesis as a valid "confusion-preventing" syntax.
```

Comment:
This should mention that its true by default

Reply:
Fixed [`ee909e5` (#54)](https://github.com/RDambrosio016/RSLint/pull/54/commits/ee909e5f37e179fdadbd498680de39aa3473b149)